ProjectK.Business/ExperiencePoints/Experience.cs

```diff
@@ -0,0 +1,61 @@
+﻿using ProjectK.Contracts;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ProjectK.Business.ExperiencePoints
+{
+    public interface IExperienceEdit
```

Comment:
You're using Fluent API pattern wrong. Please refer to `Character.cs`

Reply:
I don't remember how to properly use Fluent API

Reply:
That's why I said "Please refer to `Character.cs`"